rpm/python-scitokens-credmon.spec

```
@@ -41,18 +43,23 @@ rm -rf %{pypi_name}.egg-info
 
 %install
 %py2_install
-cp %{buildroot}/%{_bindir}/condor_credmon %{buildroot}/%{_bindir}/condor_credmon-2
-ln -sf %{_bindir}/condor_credmon-2 %{buildroot}/%{_bindir}/condor_credmon-%{python2_version}
 
 
 %files -n python2-%{pypi_name} 
 %doc 
-%{_bindir}/condor_credmon
-%{_bindir}/condor_credmon-2
-%{_bindir}/condor_credmon-%{python2_version}
+%{_bindir}/scitokens_credmon
+%{_bindir}/scitokens_credential_producer
 %{python2_sitelib}/credmon
 %{python2_sitelib}/scitokens_credmon-%{version}-py?.?.egg-info
+%attr(0750, root, condor) /var/lib/condor/credential
```

```suggestion
%attr(2770, root, condor) /var/lib/condor/credential
```
Enable the `OAuthCredmonWebserver` to write to the credential dir (and enable SetGID).